src/components/cursor.js

```diff
@@ -223,7 +225,16 @@ module.exports.Component = registerComponent('cursor', {
       mouse.x = (left / bounds.width) * 2 - 1;
       mouse.y = -(top / bounds.height) * 2 + 1;
 
-      if (camera && camera.isPerspectiveCamera) {
+      if (this.data.rayOrigin === 'xrselect' && evt.type === 'selectstart') {
+        const frame = evt.frame;
```

Comment:
use var for consistency with the code base

Reply:
How is `mousemove` triggered in an `xrselect` context?

Reply:
Currently I am just handling mousedown and mouseup but I can add mousemove, there are two option that jump to mind:

* only mousemove on an input when it is between a selectstart and selectend
* mousemove all the time for all inputs (obviously transient inputs will only mouse move when an input is pressed because they only exist when the user is touching the screen)

Reply:
My mistake here was that I forgot that the raycaster updates eah frame automatically, adding this support was a lot easier than I had expected